lib/Form/Util/Styles.php

```diff
@@ -10,6 +10,9 @@ class Styles {
 .mailpoet_form {
   padding: 20px;
 }
+.mailpoet_form.mailpoet_form_slide_in {
+  max-width: 350px;
```

Comment:
You are using custom CSS for those changes. Rosta and I have been adding our changes to `public.scss` and I am not sure which is better. I feel like the `public.scss` is slightly better because we can later change it or add settings to control those values. If a user saves a form with this custom CSS it would be more difficult for us to override those values.

Reply:
Sure I can move this to `public.scss`, which I did, increasing the range between `min-width` and `max-width` a bit. But I still have some doubts: this probably needs to be somewhat customizable, the only way is to set the `width` custom CSS property which will still make it harder to override. Then, it may still be overridden by a theme. Therefore it's still not a perfect solution, but I have no better idea.

Reply:
I'm currently working on form width settings https://mailpoet.atlassian.net/browse/MAILPOET-2811
In this feature, each form will have defined width and user will be able to set their own.